coverage/src/main/kotlin/org/rust/coverage/GrcovRunner.kt

```diff
@@ -86,8 +86,7 @@ class GrcovRunner : RsDefaultProgramRunnerBase() {
             val environmentVariables = EnvironmentVariablesData.create(
                 oldVariables.envs + mapOf(
                     "CARGO_INCREMENTAL" to "0",
-                    "RUSTFLAGS" to "-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests -Cpanic=abort",
-                    "RUSTDOCFLAGS" to "-Cpanic=abort"
+                    "RUSTFLAGS" to "-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off"
```

Comment:
How does it affect users except fix proc-macro lib tests?

Reply:
Yes, now user can catch panic. Also, previously we incorrectly showed coverage in some cases:

* before:
<img src="https://user-images.githubusercontent.com/6079006/94584610-87602e00-0287-11eb-9d2f-8167bd3dd142.png" height="200"> <img src="https://user-images.githubusercontent.com/6079006/94584650-97780d80-0287-11eb-8363-0a94101bd617.png" height="200">

* after:
<img src="https://user-images.githubusercontent.com/6079006/94638975-c91ec200-02e3-11eb-947c-545a2e1cd7a4.png" height="200"> <img src="https://user-images.githubusercontent.com/6079006/94638961-bdcb9680-02e3-11eb-9d1a-b54bc98989c6.png" height="200">

Reply:
Thanks!